rest_framework-stubs/compat.pyi

```diff
@@ -1,51 +1,56 @@
 from typing import Any
 
 from django.db.models import QuerySet
+from typing_extensions import TypeAlias
 
 try:
     from django.contrib.postgres import fields as postgres_fields
 except ImportError:
-    postgres_fields = None  # type: ignore
+    postgres_fields: TypeAlias = None  # type: ignore[no-redef]
 try:
     import coreapi
 except ImportError:
-    coreapi = None
+    coreapi: TypeAlias = None  # type: ignore[no-redef]
 try:
     import uritemplate
 except ImportError:
-    uritemplate = None  # type: ignore
+    uritemplate: TypeAlias = None  # type: ignore[no-redef]
 try:
     import coreschema
 except ImportError:
-    coreschema = None
+    coreschema: TypeAlias = None  # type: ignore[no-redef]
 try:
     import yaml
 except ImportError:
-    yaml = None  # type: ignore
+    yaml: TypeAlias = None  # type: ignore[no-redef]
 try:
     import requests
 except ImportError:
-    requests = None  # type: ignore
+    requests: TypeAlias = None  # type: ignore[no-redef]
 try:
     import pygments
 except ImportError:
-    pygments = None
+    pygments: TypeAlias = None  # type: ignore[no-redef]
 try:
     import markdown
     def apply_markdown(text: str) -> str: ...
 
 except ImportError:
-    apply_markdown = None  # type: ignore
-    markdown = None  # type: ignore
+    apply_markdown: TypeAlias = None  # type: ignore[no-redef]
+    markdown: TypeAlias = None  # type: ignore[no-redef]
 
-if markdown is not None and pygments is not None:
+try:
+    import pygments
```

Comment:
Why do we need `import pygments` here? I seems to be not used and not re-exported (if this was the goal) 🤔

Reply:
We also have it here: https://github.com/typeddjango/djangorestframework-stubs/pull/496/files#diff-eca906d3868175be921f64f15c140de74981da18a478ae18da601a43a969deb8R31

Reply:
It is re-exported in `__all__`

```python
__all__ = [
    ...
    "pygments",
```

Reply:
I've simplified this, merged it into the `try: import markdown` block.